Makefile

```diff
@@ -173,6 +173,13 @@ DATAPLANE_BRANCH     ?= main
 # Ceph
 CEPH_IMG       ?= quay.io/ceph/demo:latest
 
+# Manila
+MANILA_IMG       ?= quay.io/openstack-k8s-operators/manila-operator-index:latest
```

Comment:
This is still not present in the `quay.io/openstack-k8s-operators` namespace

Reply:
do you want to wait until that change is present in the namespace before merging?

Reply:
@dprince should we `make catalog` and `make catalog-push` to `quay.io/openstack-k8s-operators` ? I think I don't have enough permissions to push it, but apart from that, it should be ready to go.

Reply:
Fixed by [1] and now we have all we need.

https://quay.io/repository/openstack-k8s-operators/manila-operator-bundle?tab=tags&tag=latest
https://quay.io/repository/openstack-k8s-operators/manila-operator-index?tab=tags&tag=latest
https://quay.io/repository/openstack-k8s-operators/manila-operator?tab=tags&tag=latest

[1] https://github.com/openstack-k8s-operators/manila-operator/pull/37